src/Workspaces/Core/Portable/Log/HistogramLogAggregator.cs

```diff
@@ -86,9 +92,9 @@ public string GetBucketsAsString()
                 return pooledStringBuilder.ToStringAndFree();
             }
 
-            private int GetBucket(decimal value)
+            private int GetBucket(int value)
```

Comment:
In practice, we were always converting ints to decimals to pass to here, then doing some decimal math only to convert back to an int. This just keeps it simpler.